packages/workflows/__tests__/__system__/Archive.system.test.ts

```diff
@@ -0,0 +1,204 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Session, Imperative } from "@brightside/imperative";
+import { IArchivedWorkflow } from "../../src/api/doc/IArchivedWorkflow";
+import { ArchiveWorkflow } from "../..";
+import { WorkflowConstants } from "../../src/api/WorkflowConstants";
+import { ITestEnvironment } from "../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { TestProperties } from "../../../../__tests__/__src__/properties/TestProperties";
+import { TestEnvironment } from "../../../../__tests__/__src__/environment/TestEnvironment";
+import { Upload } from "../../../zosfiles/src/api";
+import { CreateWorkflow } from "../../src/api/Create";
+import { ZosmfRestClient } from "../../../rest";
+
+let session: Session;
+let testEnvironment: ITestEnvironment;
+const workflowKeyConst: string = "0123-456789-abc-def";
+let workflowKeyActual: string;
+const localWorkflowPath: string = `${__dirname}/testfiles/demo.xml`;
+let remoteWorkflowPath: string;
+let sysProperties: TestProperties;
+const allWorkflowKeys: string[] = [];
+
+async function setup() {
+    testEnvironment = await TestEnvironment.setUp({
+        testName: "archive_workflow"
+    });
+    sysProperties = new TestProperties(testEnvironment.systemTestProperties);
+    session = TestEnvironment.createZosmfSession(testEnvironment);
+    remoteWorkflowPath=`${sysProperties.getDefaultSystem().unix.testdir}/wf${Date.now()}.xml`;
+}
+
+async function cleanup() {
+    await TestEnvironment.cleanUp(testEnvironment);
+}
+
+async function removeWorkflows() {
+    while(allWorkflowKeys.length > 0) {
+        const currentKey = allWorkflowKeys.pop();
+        const query: string = `/zosmf/workflow/rest/1.0/archivedworkflows/${currentKey}`;
+        const deleted = await ZosmfRestClient.deleteExpectString(session, query);
+        Imperative.console.info(JSON.stringify(deleted));
+    }
+}
+
+describe("Archive workflow unit tests - successful scenarios", () => {
+    beforeAll(async ()=> {
+        await setup();
+        await Upload.fileToUSSFile(session, localWorkflowPath, remoteWorkflowPath, true);
+    });
+    beforeEach(async ()=> {
+        const systemName = testEnvironment.systemTestProperties.systems.primary;
```

Comment:
Could you use the `workflows.system` property instead of this one? I might go through the properties file and remove unused fields and simplify the file a bit soon.

Reply:
Hi Chris, I will check it tomorrow. Thanks for the suggestion :)